lib/recipeData.tsx

```diff
@@ -137,29 +128,31 @@ const trigger = useCallback(async () => {
     details:
       'Build an email/password authentication experience including passwords resets, password strength checking, and magic links using prebuilt Stytch UI components.',
     description: ``,
-    instructions: `To the right you'll the Stytch UI configured for email/password login. Enter a new email address and you will be prompted to create an account with a secure password.`,
+    instructions: `To the right you'll the Stytch UI configured for password login. Enter a new email address and you will be prompted to create an account with a secure password.`,
```

Comment:
Noticed we are missing a verb

"to the right you'll ???..."

Reply:
[b121628](https://github.com/stytchauth/stytch-nextjs-integration/pull/125/commits/b1216284621a5a1a6da9ebe6be1b8fcb22db7850)